extensions/wikia/UserProfilePageV3/templates/UserProfilePage_renderLightbox.php

```diff
@@ -13,31 +13,25 @@
 
 	<ul class="tab-content">
 		<li class="avatar">
-			<form id="usersAvatar" class="WikiaForm" name="usersAvatar" method="post" enctype="multipart/form-data"
-				action="<?= UserProfilePageController::getLocalUrl('onSubmitUsersAvatar', [ 'userId' => $userId ], 'json'); ?>">
+			<form id="avatar-upload-form" class="WikiaForm" method="post" enctype="multipart/form-data">
 
 				<?= $avatar; ?>
 
 				<fieldset class="avatar-options">
-					<? if ( $isUploadsPossible ): ?>
-						<div class="input-group">
-							<label for="UPPLightboxAvatar">
-								<button id="UPPLightboxAvatarUpload"><?= wfMessage( 'user-identity-box-avatar-upload-avatar' )->escaped(); ?></button>
-							</label>
-							<input type="file" name="UPPLightboxAvatar" id="UPPLightboxAvatar">
-							<input type="hidden" name="UPPLightboxDefaultAvatar" id="UPPLightboxDefaultAvatar" value="">
-							<input type="hidden" name="UPPLightboxFbAvatar" id="UPPLightboxFbAvatar" value="">
-							<input type="hidden" name="MAX_FILE_SIZE" value="<?= $avatarMaxSize; ?>">
-						</div>
-					<? endif; ?>
+					<div class="input-group">
+						<label for="avatar-upload-input">
+							<?= wfMessage( 'user-identity-box-avatar-upload-avatar' )->escaped(); ?>
+						</label>
+						<input type="file" name="file" id="avatar-upload-input" accept="image/*" />
```

Comment:
We want to limit the types of files that can be uploaded to these, right? Maybe we could enumerate those [here](https://github.com/Wikia/app/pull/16081/files?utf8=%E2%9C%93&diff=split#diff-24765cebf3960a2bf8cb16081c7af852L24) in the `accept` attribute?

Reply:
Yea, that's a fair point. This approach makes sense to me 👍